src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -1337,6 +1337,19 @@ PUBLIC_API STATUS peerConnectionGetCurrentLocalDescription(PRtcPeerConnection, P
  */
 PUBLIC_API STATUS createOffer(PRtcPeerConnection, PRtcSessionDescriptionInit);
 
+/**
+ * @brief The canTrickleIceCandidates attribute indicates whether the remote peer is able to accept trickled ICE candidates.
+ * The value is determined based on whether a remote description indicates support for trickle ICE. Prior to the completion
+ * of setRemoteDescription, this value is null.
+ *
+ * Reference: https://www.w3.org/TR/webrtc/#dom-rtcpeerconnection-cantrickleicecandidates
+ *
+ * @param[in] PRtcPeerConnection Initialized RtcPeerConnection
+ *
+ * @return NullableBool if not null, indicate whether remote support trickle ICE.
+ */
+PUBLIC_API NullableBool canTrickleIceCandidates(PRtcPeerConnection);
```

Comment:
NullableBool is a structure and not an intrinsic type. Returning a structure is by copy is not a good idea in functions - in fact, if you search the entire codebase we have no occurrences. I recommend taking in a pointer and setting it perhaps.

Reply:
these are small structures whats bad about copying them? Im trying to stick to the public interfaces in w3c.

Reply:
It shouldn't be taken verbatim in my opinion. I would prefer the functions not to return entire structures. Though, in this case, I see your point.